guides/technical-overview.mdx

```diff
@@ -0,0 +1,45 @@
+---
+order: 5
+title: Technical Overview
+description: Explaining technical decisions in the AgDS Component library
+---
+
+## React
+
+The AgDS Component library is built using React. React is a JavaScript library for building user interfaces. It is maintained by Facebook and a community of individual developers and companies.
+
+React uses a component-based model for defining UI elements, which is fantastic for building Design Systems and scalable UI generally.
+
+## NPM Packages
+
+The delivery method for the AgDS Component library is via NPM packages. NPM is the package manager for JavaScript and the world’s largest software registry.
+
+Each component is an individual package. This allows consumers to only install the components they need, and avoid installing unnecessary dependencies and huge bundles for users.
+
+```zsh
```

Comment:
I don't think this example is necessary. Or if you think it is, can you add a H3 above it?